### PR TITLE
Add MutePad keyboard (Handwired)

### PR DESCRIPTION
**Pull request by JoshwJB**
## Description
Add MutePad keyboard (Handwired) with 3D printable case - see [MutePad Repo](https://github.com/JoshwJB/MutePad)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mutepad/config.h [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+/* encoder support */
+#define ENCODERS_PAD_A \
+    { F4 }
+#define ENCODERS_PAD_B \
+    { F5 }
+#define ENCODERS_PAD_A_RIGHT \
+    { F5 }
+#define ENCODERS_PAD_B_RIGHT \
+    { F4 }
```
Comment (zvecr) [inline keyboards/mutepad/config.h:43]:
```suggestion
```

File: keyboards/mutepad/readme.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# MutePad
+
+![MutePad Repo](https://github.com/JoshwJB/MutePad)
+
+*A short description of the keyboard/project*
```
Comment (zvecr) [inline keyboards/mutepad/readme.md:5]: Needs to be filled in, or removed

**Review by zvecr — CHANGES_REQUESTED** [review]
Needs to be moved to `keyboards/handwired/mutepad`.

File: keyboards/mutepad/readme.md (hunk 2 above)
Comment (JoshwJB) [reply keyboards/mutepad/readme.md]: Should be updated now. Thanks for taking a look.

File: keyboards/handwired/mutepad/readme.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# MutePad
+
+![MutePad Repo](https://github.com/JoshwJB/MutePad)
```
Comment (fauxpark) [inline keyboards/handwired/mutepad/readme.md:3]: This syntax is for an image, not a link.

File: keyboards/handwired/mutepad/readme.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+*A 3D printed under desk mounted keypad with rotary encoder, for muting applications (discord, zoom etc.) and adjusting volume.*
```
Comment (fauxpark) [inline keyboards/handwired/mutepad/readme.md:5]:
```suggestion
A 3D printed under desk mounted keypad with rotary encoder, for muting applications (discord, zoom etc.) and adjusting volume.
```

File: keyboards/handwired/mutepad/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+// Defines names for use in layer keycodes and the keymap
+enum layer_names { _BASE };
+
+enum custom_keycodes { MUTE_GOOGLE_MEET = 0, MUTE_TEAMS = 1 };
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case MUTE_GOOGLE_MEET:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LCMD("d"));
```
Comment (fauxpark) [inline keyboards/handwired/mutepad/keymaps/default/keymap.c:15]:
```suggestion
                tap_code16(G(KC_D));
```

File: keyboards/handwired/mutepad/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            }
+            break;
+        case MUTE_TEAMS:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LCTRL(SS_LSFT("m")));
```
Comment (fauxpark) [inline keyboards/handwired/mutepad/keymaps/default/keymap.c:20]:
```suggestion
                tap_code16(C(S(KC_M)));
```

File: keyboards/handwired/mutepad/config.h [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
```
Comment (fauxpark) [inline keyboards/handwired/mutepad/config.h:91]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/mutepad/readme.md (hunk 3 above)
Comment (JoshwJB) [reply keyboards/handwired/mutepad/readme.md]: Updated now. Thanks for taking a look.

**Comment by JoshwJB** [comment]
Should be good for a re-review now.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**